functions-app/calculate-score-v2/check-window.data.service.js

```diff
@@ -0,0 +1,17 @@
+const sqlService = require('../lib/sql/sql.service')
+const R = require('ramda')
+
+const schema = '[mtc_admin]'
+const checkWindowTable = '[checkWindow]'
+
+/**
+ * Find a check window within the score calculation period
+ * @return {Object}
+ */
+module.exports.sqlFindCalculationPeriodCheckWindow = async () => {
+  const sql = `
+  SELECT * from ${schema}.${checkWindowTable}
+  WHERE GETUTCDATE() BETWEEN checkStartDate AND adminEndDate`
```

Comment:
how do we calculate scores outside of the check window? is that a possibility?  if so, accommodate the 'last' check window

Reply:
awaiting feedback on check window end date vs admin end date

Reply:
@GuyHarwood is this resolved then?

Reply:
yes